src/map/zone.h

```diff
@@ -546,6 +547,9 @@ class CZone
     void   SetLocalVar(const char* var, uint32 val);
     void   ResetLocalVars();
 
+    /// Finds all npcs matching the given pattern. Patterns can contain % as wildcard characters to
+    /// match incomplete names. e.g: %anto% matches Shantotto and Canto-Ranto
+    virtual const NPCEntityList_t GetNPCsByName(std::string const& pattern);
```

Comment:
Like with function args, `const` goes on the right side of the type, making it clearer at a glance what exactly is const (more useful when fighting against `type& const` and `type const&` etc.)

Reply:
Done